R/tab_options.R

```diff
@@ -230,6 +244,42 @@ tab_options <- function(data,
     opts_df <- opts_df_set(opts_df, "table_width", table.width)
   }
 
+  # table.align
+  if (!is.null(table.align)) {
+
+    if (table.align == "center") {
+      opts_df <- opts_df_set(opts_df, "margin_left", "auto")
```

Comment:
being able to set a `margin_left` / right but not being able to set it directly seems off to me.

Reply:
I can't think of a _nice_ solution without throwing warning messages if certain values have already been set.  Almost as if `table.align` is a _super_ value that will set multiple table options.  If some of those table options have already been set, maybe display a warning?  

But that solution seems to not be aligned with ggplot2 theme ideas.  Hmmm

Reply:
The way it's set up now is to have `table.align` stomp over `margin_left` and `margin_right`. I can't help but wonder that there should be a `table_align` setting that stored and takes precedence over any setting of `margin_left` or `margin_right`.